### PR TITLE
boost repo mirror

### PR DESCRIPTION
**Pull request by ArqTras**
boost repository mirror https://boostorg.jfrog.io/artifactory/main

### CODE REVIEW

**Comment by BigslimVdub** [comment]
curious as to why not push commits to #297 since it will need to be rebased to this anyway or are you just bumping to master for the current build requirements?

File: contrib/depends/packages/boost.mk [hunk 1]
```diff
@@ -1,6 +1,6 @@
 package=boost
 $(package)_version=1_73_0
```
Comment (malbit) [inline contrib/depends/packages/boost.mk:2]: consider to use "version_dot" style instead

File: contrib/depends/packages/boost.mk [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
-$(package)_download_path=https://gitlab.com/ArqTras/depends/-/raw/main/
+$(package)_download_path=https://boostorg.jfrog.io/artifactory/main/release/1.73.0/source/
```
Comment (malbit) [inline contrib/depends/packages/boost.mk:3]: "1.73.0" can be replaced with: "$($(package)_version)"

File: contrib/depends/packages/boost.mk [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
 $(package)_file_name=boost_$($(package)_version).tar.bz2
```
Comment (malbit) [inline contrib/depends/packages/boost.mk:4]: Along with both changes mention above "file_name" can be replaced with: "boost_$(subst .,_,$($(package)_version))" which will be more future_changes compatible and will require only to change package version number and sha256 hash :)

**Review by malbit — COMMENTED** [review]
Have a look at my comments and if you are ok with them do changes or if not then don't

**Comment by malbit** [comment]
> curious as to why not push commits to #297 since it will need to be rebased to this anyway or are you just bumping to master for the current build requirements?

Not every #PR pushed will require rebase. If no cross-files have changes at both #PR's it is ok and safe to merge with separate commits and will not require rebase

**Review by malbit — APPROVED** [review]

**Pull request merged**